util/util.go

```diff
@@ -1014,3 +1014,19 @@ func VerifySnapshotLabels(labels map[string]string) error {
 	}
 	return nil
 }
+
+func IsTagsEqual(tags1, tags2 []string) bool {
```

Comment:
Can we have a doc comment for this function? Suggest taking any change to improve this basic coding practice. (also, unit tests will be good to have for this exporeted function)